package.json

```diff
@@ -2,7 +2,7 @@
   "name": "branch-cordova-sdk",
   "description": "Branch Metrics Cordova SDK",
   "main": "src/index.js",
-  "version": "3.3.0",
+  "version": "3.3.1",
```

Comment:
Maybe bump the minor version since the library dependencies have changed?  Though on cordova this should be transparent to users.

Reply:
Ok, good thinking I'll do that